mlflow/tracking/default_experiment/databricks_job_experiment_provider.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from mlflow.tracking.default_experiment.abstract_context import DefaultExperimentProvider
+from mlflow.utils import databricks_utils
+from mlflow.tracking.client import MlflowClient
+from mlflow.utils.mlflow_tags import (
+    MLFLOW_DATABRICKS_JOB_TYPE_INFO,
+    MLFLOW_EXPERIMENT_SOURCE_TYPE,
+    MLFLOW_EXPERIMENT_SOURCE_ID,
+)
+
+_logger = logging.getLogger(__name__)
+_active_job_experiment_id = None
```

Comment:
We don't necessarily know if it's active or not (the user might have set a different experiment manually).

```suggestion
_resolved_job_experiment_id = None
```